entity-service/src/integrationTest/java/org/hypertrace/entity/service/service/EntityQueryServiceTest.java

```diff
@@ -981,6 +997,194 @@ void testExecuteWithExistsFilter() {
     assertTrue(list.get(1).getResultSetMetadata().getColumnMetadataCount() > 0);
   }
 
+  @Test
```

Comment:
Does the test run against both the DB? Or Did we try manually against Postgres?

Reply:
@aman Yes, the test runs agains only MongoDB. Can we grep the translated docStore query and see if its compatible? or Can you paste the translated query here?